src/Cake/Scripting/Roslyn/Stable/RoslynScriptSessionFactory.cs

```diff
@@ -10,15 +10,15 @@ namespace Cake.Scripting.Roslyn.Stable
 {
     using Core.IO;
```

Comment:
@devlead @patriksvensson I am going to guess that this is ReSharper and StyleCop getting in the way of each other.  By default, StyleCop wants the usings within the namespace.  As a result, if you do an ALT + ENTER, depending on your setup, it will add any "new" using statements within the namespace.  This happens to me all the time, and most times I remember to fix it.  I think there should be a setting we could add to the ReSharper settings file for the project to specify that usings are outside the namespace.